docs/content/preview/yugabyte-cloud/cloud-monitor/logging-export.md

```diff
@@ -129,7 +130,6 @@ Log all connection attempts, along with successfully completed client authentica
 
 Log session termination and duration of the session.
 
-<!--
 ## Database Audit Log
```

Comment:
```suggestion
## Database Audit Logging
```

Also replace `Database Audit Log` -> `Database Audit Logging` in steps.

`YSQL audit log settings` -> `YSQL audit logging settings`